lib/sidekiq.rb

```diff
@@ -127,9 +129,13 @@ def self.logger=(log)
     Sidekiq::Logging.logger = log
   end
 
+  # When set, overrides Sidekiq.options[:global_poll_interval_average] and sets the
+  # average interval that this process will delay before checking for scheduled jobs
+  # or job retries that are ready to run.
+  #
   # See sidekiq/scheduled.rb for an in-depth explanation of this value
   def self.poll_interval=(interval)
```

Comment:
This needs to print a deprecated warning.  The only thing that should be exposed in Sidekiq 4.0+ is global_poll_interval_average (which still needs a better name).